react/react1/week2/todo-list/src/AddTodo.jsx

```diff
@@ -0,0 +1,31 @@
+import React,{ useState}  from 'react';
+
+import TodoList from "./List";
+import todoList  from './data';
+<todoList todoList = {todoList}/>
+
+export default function AddTodo () { 
+  const  [list,setList] = useState(todoList);
+
+  const addTaskClick= ()=> {
```

Comment:
are you using naming or todo?

Reply:
I don't understand what you mean, naming or todo?